### PR TITLE
{tools}[GCCcore/12.2.0] nvtop v3.0.1

### PR DESCRIPTION
**Pull request by anselmicz**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
torrent - Linux Debian GNU/Linux 12, x86_64, AMD Ryzen 5 3600 6-Core Processor (zen2), 1 x NVIDIA NVIDIA GeForce RTX 3070, 525.89.02, Python 3.11.2
See https://gist.github.com/anselmicz/76177eadc70ce99b5af82bf5f9e603a7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0203u29a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), 1 x NVIDIA NVIDIA A100-SXM4-80GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/e764f9e935ab93fa3bfb9559f071cd03 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
this needs some `osdependencies`, see the README

redhat-based:
libdrm-devel systemd-devel

ubuntu:
libdrm-dev libsystemd-dev (libudev-dev)

File: easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['f61ecbe3a30a216706c16fc177d3b840e7d7deccf87b048d59f95f230d591a53']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+]
+
```
Comment (anselmicz) [inline easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb:19]:
```suggestion

osdependencies = [('libsystemd-dev', 'systemd-devel')]

```

Comment (anselmicz) [reply easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb:19]: Shouldn't `libsystemd-dev` and `systemd-devel` be enough considering we can build `libdrm` ourselves?

File: easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Note: software dlopen's libraries from the OS installed drivers, thus no specific CUDA dependency.
+dependencies = [
+    ('ncurses', '6.3'),
+    ('libdrm', '2.4.114'),
+]
+
```
Comment (anselmicz) [inline easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb:25]:
```suggestion

osdependencies = [('libsystemd-dev', 'systemd-devel')]

```

Shoudn't `libsystemd-dev` and `systemd-devel` be enough considering we build `libdrm` ourselves?

Comment (smoors) [reply easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb:25]: indeed, you're right

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
torrent - Linux Debian GNU/Linux 12, x86_64, AMD Ryzen 5 3600 6-Core Processor (zen2), 1 x NVIDIA NVIDIA GeForce RTX 3070, 525.105.17, Python 3.11.2
See https://gist.github.com/anselmicz/1ec907a99d6a659b52dc828413637056 for a full test report.

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
torrent - Linux Debian GNU/Linux 12, x86_64, AMD Ryzen 5 3600 6-Core Processor (zen2), 1 x NVIDIA NVIDIA GeForce RTX 3070, 525.105.17, Python 3.11.2
See https://gist.github.com/anselmicz/39757782a06fb07672257471df35d02f for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 17 out of 17 (1 easyconfigs in total)
alvis-s1 - Linux Rocky Linux 8.6, x86_64, Intel(R) Xeon(R) Silver 4314 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/Micket/1c657a93416bd9230f51dd762c4e2ef0 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 17 out of 17 (1 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/739dcf45b26b4cef2d2495f18f1089f1 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @anselmicz!

**Pull request merged**